### DISCUSSION TITLE
Wrong version number when building vips-8-14.1 from source

### DISCUSSION THREAD

**Discussion by z26100** [libvips/libvips#3364] [General]
![image](https://user-images.githubusercontent.com/4007385/221624061-e6d752c3-bb7d-4592-bd02-90768349f7db.png)

**Comment by lovell** [comment]
This looks like an autotools-generated version, but libvips 8.14 uses Meson so there's probably an old/conflicting version of `vips` on this machine.

If you're still having problems, please can you provide complete details that allow someone else to reproduce, including all commands you're running.
